src/sentry/interfaces/message.py

```diff
@@ -80,3 +80,6 @@ def get_path(self):
 
     def get_hash(self):
         return [self.message]
+
+    def to_string(self, event, is_public=False, **kwargs):
+        return self.formatted or self.message
```

Comment:
This makes sure that it gets picked up here, right? https://github.com/getsentry/sentry/blob/master/src/sentry/plugins/sentry_mail/models.py#L167-L174